custom/enikshay/private_sector_datamigration/factory.py

```diff
@@ -32,9 +32,10 @@ def get_human_friendly_id():
 
 class BeneficiaryCaseFactory(object):
 
-    def __init__(self, domain, beneficiary):
+    def __init__(self, domain, beneficiary, location_owner):
```

Comment:
Is this class only ever instantiated in `create_cases_by_beneficiary.py`? If not, I think this new argument should have a default value of `None` to prevent other uses from breaking.